src/client/java/teammates/client/scripts/StatisticsPerInstitute.java

```diff
@@ -209,34 +204,33 @@ private String getInstituteForInstructors(List<Instructor> instructorList) {
         return institute;
     }
 
-    private String getInstituteForInstructor(Instructor instructor) {
+    private String getInstituteForInstructor(Instructor instructor, List<Account> allAccounts) {
```

Comment:
Can we use a `HashMap` to memoize this too, just like in `getInstituteForStudent`?